configure.ac

```diff
@@ -1133,6 +1133,18 @@ AS_CASE([$PY_SUPPORT_TIER],
 
 AC_DEFINE_UNQUOTED([PY_SUPPORT_TIER], [$PY_SUPPORT_TIER], [PEP 11 Support tier (1, 2, 3 or 0 for unsupported)])
 
+AS_CASE([$host],
+  [aarch64*], [
+    AS_FOR(option, opt, [-mbranch-protection=pac-ret -msign-return-address=all], [
```

Comment:
I wonder if we should write out the `for` loop instead. AFAIK, `AS_FOR` is undocumented.